lib/GLPI/Agent/Task/Inventory/Generic/Remote_Mgmt/RustDesk.pm

```diff
@@ -37,7 +37,23 @@ sub doInventory {
     # Add support for --get-id parameter available since RustDesk 1.2 as id becomes empty in conf
     # Only works starting with RustDesk v1.2.2
     unless (defined($RustDeskID) && length($RustDeskID)) {
-        my $command = OSNAME eq 'MSWin32' ? 'C:\Program Files\RustDesk\rustdesk.exe' : 'rustdesk';
+        my $command = 'rustdesk';
+        if(OSNAME eq 'MSWin32'){
+            my $Registry;
+            my $installLocation;
+            Win32::TieRegistry->require();
+            Win32::TieRegistry->import(
+                Delimiter   => '/',
+                ArrayValues => 0,
+                TiedRef     => \$Registry
+            );
+
+            my $key = $Registry->Open('LMachine/SOFTWARE/Microsoft/Windows/CurrentVersion/Uninstall/RustDesk', { Access => Win32::TieRegistry::KEY_READ() });
+
+            if(defined($key)){ $installLocation = $key->GetValue('InstallLocation') . '\rustdesk.exe'; }
+
+            $command = (defined($installLocation) && length($installLocation)) ? $installLocation : 'C:\Program Files\RustDesk\rustdesk.exe';
```

Comment:
You don't have to load Win32::TieRegistry as GLPI::Agent::Tools::Win32 still provides an API for your purpose.

```suggestion
            GLPI::Agent::Tools::Win32->require();
            my $installLocation = GLPI::Agent::Tools::Win32::getRegistryValue(
                path   => "SOFTWARE/Microsoft/Windows/CurrentVersion/Uninstall/RustDesk/InstallLocation",
                logger => $logger
            );
            $command = (empty($installLocation) ? 'C:\Program Files\RustDesk' :$installLocation) . '\rustdesk.exe';
```

`empty()` is also a provided API in GLPI::Agent::Tools to simplify the related test.

Can you test this version and better use it for your PR if it works as expected ?